app/src/main/java/com/example/mymovieinfo/dto/Movie.kt

```diff
@@ -2,12 +2,14 @@ package com.example.mymovieinfo.dto
 
 import com.google.gson.annotations.SerializedName
 
-data class Movie(@SerializedName("rank")var rank: Int, @SerializedName("title")var title: String,
-                 @SerializedName("country") var country: String, @SerializedName("boxOfficeGross")var boxOfficeGross: Double,
-                 @SerializedName("openingWeekendGross")var openingWeekendGross: Double, @SerializedName("distributor")var distributor: String){
+data class Movie(@SerializedName("rank") var rank: Int,
+                 @SerializedName("title") var title: String,
+                 @SerializedName("country") var country: String,
+                 @SerializedName("boxOfficeGross") var boxOfficeGross: Double,
+                 @SerializedName("openingWeekendGross") var openingWeekendGross: Double,
+                 @SerializedName("distributor") var distributor: String) {
 
```

Comment:
Restructured the dto according to kotlin coding conventions: https://kotlinlang.org/docs/coding-conventions.html